fe/fe-core/src/main/java/org/apache/doris/qe/Coordinator.java

```diff
@@ -837,7 +837,10 @@ private void computeFragmentExecParams() throws Exception {
                 int bucketSeq = 0;
                 int bucketNum = bucketShuffleJoinController.getFragmentBucketNum(destFragment.getFragmentId());
                 TNetworkAddress dummyServer = new TNetworkAddress("0.0.0.0", 0);
-
+                if (destParams.instanceExecParams.size() == 1 && destParams.instanceExecParams.get(0).bucketSeqSet.isEmpty()) {
```

Comment:
Pleaase add some comment here，It means left table is empty(). To correct bucket shuffle join, so need give a fake bucekt num to help left instance close() right. Thank you !